### PR TITLE
Create zip for Netsuite, create 2 reports on call

### PR DESCRIPTION
**Pull request by jason-jackson**
Alternate PR to #518, I wanted to see if this looked any cleaner. 

### Changed
- Create Ledger report makes 2 reports, both Sage CSV and NetSuite Zip file
- Added more information to Readme about `cover.local`

### CODE REVIEW

File: application/actions/ledger.go [hunk 1]
```diff
@@ -114,6 +111,31 @@ func ledgerReportCreate(c buffalo.Context) error {
 		return reportError(c, err)
 	}
 
+	// Create NetSuite report
```
Comment (jason-jackson) [inline application/actions/ledger.go:114]: I'm debating adding a `NewLedgerReport` function to LedgerEntries that would move the bulk of this out of here.

File: README.md
Comment (jason-jackson) [inline README.md:1]: Looks like formatter kicked in, I could revert and try again if needed.

**Comment by jason-jackson** [comment]
Pros:

- Don't need to modify database
- Don't need to modify UI (potentially, need to check if UI cares about return of Ledger Report Create)
- Easy to switch if we change providers later on

**Review by briskt — APPROVED** [review]
This does look pretty good. You may want to add a column to the database to store the report format. The content type (csv/zip) can help, but it's not quite the same. Not sure whether it would go on the API response or not, I guess that's up to the UI/UX people.

**Comment by jason-jackson** [comment]
The file name also has the report format, so that could be used as well.

**Pull request merged**